docs/site/BelongsTo-relation.md

```diff
@@ -75,18 +75,51 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
+
 The definition of the `belongsTo` relation is inferred by using the `@belongsTo`
 decorator. The decorator takes in a function resolving the target model class
 constructor and designates the relation type. It also calls `property()` to
 ensure that the decorated property is correctly defined.
+Usage of belongsTo relation is similar to legacy datasource juggler.
+
+The `@belongsTo` decorator takes three parameters :
+  - the `target model class`,
+  - the `relation definition` - has three attributes, keyFrom, keyTo, name
+    - `keyFrom` is a property name on the "source" model (e.g. Order.customerId). keyFrom attribute has become a dummy parameter and is not relevant any more with the recent changes to use the decorated target as the actual keyFrom value.
+    - `keyTo` is a property name on the "target" model, typically the primary key of the "target" model.
+    - `name` is the relation name.
+  - the `property definition`, this creates a property decorator implicitly. The name attribute in the definition can be used to customize datasource column name.
 
 A usage of the decorator with a custom primary key of the target model for the
 above example is as follows:
 
 ```ts
 class Order extends Entity {
   // constructor, properties, etc.
-  @belongsTo(() => Customer, {keyTo: 'pk'})
+  @belongsTo(
+    () => Customer,
+    { keyFrom: 'customerId', keyTo: 'id', name: 'customer' },
```

Comment:
I still don't know why we are using this example. 
The community use this example because he "needed the property name in my model class to be camel-cased but the db column name to be snake-cased". That's why he used the third parameter.
But I don't think this scenario fits the scope of this task. 

We wanted to explain about customizing foreign key names. But in your example, I don't see any usage of customizing the foreign key name cause it's still `customerId`.

Reply:
agreed

Reply:
example to modify foreign key property should be added. could you please check PR https://github.com/strongloop/loopback-next/pull/3808/files . Opened new PR because travis jobs for this PR has some errors.